src/huggingface_hub/hf_file_system.py

```diff
@@ -591,6 +592,42 @@ def url(self, path: str) -> str:
             url = url.replace("/resolve/", "/tree/", 1)
         return url
 
+    def get_file(self, rpath, lpath, callback=_DEFAULT_CALLBACK, outfile=None, **kwargs):
```

Comment:
We should parse the `kwargs` (to propagate `revision` when present to `self.resolve_path`/`self.info`, call `super().get_file(...)` if `block_size` is specified, etc.)

Reply:
Thanks! Would have definitely missed this one :grimacing: 
Do you see other use cases? I think i'll propagate `revision` and if any other kwarg is present, default back to slow download unfortunately :confused:

Reply:
Yes, this solution sounds good.